package.json

```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "server-dev": "nodemon --inspect server/index.js",
-    "seed-database": "psql 9trails-paths < ../db/schema.sql"
+    "seed-database": "psql 9trails-paths < db/schema.sql"
```

Comment:
Soooo, I take it the `[Dont' Merge]` means just that? On line 12 you may want to perform a check of the fetzer valve before you engage the hyperdrive. Your bearings housing will also be more efficient during liftoff. Just a thought tho.